mavis/test/data/__init__.py

```diff
@@ -14,61 +15,100 @@
-    VACCS_EMPTY_FILE: Final[str] = "VACCS_HPV_EMPTY_FILE"
-    VACCS_HPV_DOSE_TWO: Final[str] = "VACCS_HPV_DOSE_TWO"
-    VACCS_HEADER_ONLY: Final[str] = "VACCS_HPV_HEADER_ONLY"
-    VACCS_MAV_853: Final[str] = "VACCS_HPV_MAV_853"
-    VACCS_HPV_MAV_855: Final[str] = "VACCS_HPV_MAV_855"
-    VACCS_SYSTMONE_POSITIVE: Final[str] = "VACCS_SYSTMONE_POSITIVE"
-    VACCS_SYSTMONE_NEGATIVE: Final[str] = "VACCS_SYSTMONE_NEGATIVE"
-    VACCS_SYSTMONE_HIST_NEGATIVE: Final[str] = "VACCS_SYSTMONE_HIST_NEGATIVE"
-    VACCS_MAV_1080: Final[str] = "VACCS_MAV_1080"
-    VACCS_SYSTMONE_MAV_1080: Final[str] = "VACCS_SYSTMONE_MAV_1080"
-    COHORTS_POSITIVE: Final[str] = "COHORTS_POSITIVE"
-    COHORTS_NEGATIVE: Final[str] = "COHORTS_NEGATIVE"
-    COHORTS_INVALID_STRUCTURE: Final[str] = "COHORTS_INVALID_STRUCTURE"
-    COHORTS_EMPTY_FILE: Final[str] = "COHORTS_EMPTY_FILE"
-    COHORTS_HEADER_ONLY: Final[str] = "COHORTS_HEADER_ONLY"
-    CHILD_POSITIVE: Final[str] = "CHILD_POSITIVE"
-    CHILD_NEGATIVE: Final[str] = "CHILD_NEGATIVE"
-    CHILD_INVALID_STRUCTURE: Final[str] = "CHILD_INVALID_STRUCTURE"
-    CHILD_EMPTY_FILE: Final[str] = "CHILD_EMPTY_FILE"
-    CHILD_HEADER_ONLY: Final[str] = "CHILD_HEADER_ONLY"
-    CHILD_MAV_1080: Final[str] = "CHILD_MAV_1080"
-    CLASS_POSITIVE: Final[str] = "CLASS_POSITIVE"
-    CLASS_NEGATIVE: Final[str] = "CLASS_NEGATIVE"
-    CLASS_INVALID_STRUCTURE: Final[str] = "CLASS_INVALID_STRUCTURE"
-    CLASS_EMPTY_FILE: Final[str] = "CLASS_EMPTY_FILE"
-    CLASS_HEADER_ONLY: Final[str] = "CLASS_HEADER_ONLY"
-    CLASS_CHILDREN_FILTER: Final[str] = "CLASS_CHILDREN_FILTER"
-    CLASS_YEAR_GROUP: Final[str] = "CLASS_YEAR_GROUP"
-    CLASS_SESSION_ID: Final[str] = "CLASS_SESSION_ID"
-    CLASS_SINGLE_VACC: Final[str] = "CLASS_SINGLE_VACC"
-    CLASS_MAV_854: Final[str] = "CLASS_MAV_854"
-    CLASS_MAV_965: Final[str] = "CLASS_MAV_965"
-    CLASS_MAV_1080: Final[str] = "CLASS_MAV_1080"
-    COHORTS_NO_CONSENT: Final[str] = "COHORTS_NO_CONSENT"
-    COHORTS_CONFLICTING_CONSENT: Final[str] = "COHORTS_CONFLICTING_CONSENT"
-    COHORTS_E2E_1: Final[str] = "COHORTS_E2E_1"
-    CLASS_MOVES_CONFIRM_IGNORE: Final[str] = "CLASS_MOVES_CONFIRM_IGNORE"
-    CLASS_MOVES_UNKNOWN_HOMESCHOOLED: Final[str] = "CLASS_MOVES_UNKNOWN_HOMESCHOOLED"
-    CLASS_CHANGE_NHSNO: Final[str] = "CLASS_CHANGE_NHSNO"
-    COHORTS_UCR_MATCH: Final[str] = "COHORTS_UCR_MATCH"
-    COHORTS_CONSENT_TWICE: Final[str] = "COHORTS_CONSENT_TWICE"
-    COHORTS_CONFLICTING_GILLICK: Final[str] = "COHORTS_CONFLICTING_GILLICK"
-    COHORTS_FULL_NAME: Final[str] = "COHORTS_FULL_NAME"
-    COHORTS_MAV_927_PERF: Final[str] = "COHORTS_MAV_927_PERF"
-    COHORTS_MAV_909: Final[str] = "COHORTS_MAV_909"
-    COHORTS_MAV_853: Final[str] = "COHORTS_MAV_853"
-    COHORTS_GILLICK_NOTES_LENGTH: Final[str] = "COHORTS_GILLICK_NOTES_LENGTH"
+class FileMapping(StrEnum):
+    @property
+    def input_template_path(self) -> str:
+        return f"{self.folder}/i_{self.value}.csv"
```

Comment:
Could we use `pathlib` where possible?

```suggestion
        return self.folder / "i_{self.value}.csv"
```

If we made `self.folder` return a `Path`?